<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="5" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="583766" started-at="2020-07-01T10:38:59Z" finished-at="2020-07-01T10:48:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="583766" started-at="2020-07-01T10:38:59Z" finished-at="2020-07-01T10:48:43Z">
      <class name="testcases.Test_Edit_Profile">
        <test-method status="PASS" signature="set()[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="set" is-config="true" duration-ms="3095" started-at="2020-07-01T17:39:00Z" finished-at="2020-07-01T17:39:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method status="FAIL" signature="edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="edit_profile_form_field_alert" duration-ms="114021" started-at="2020-07-01T17:39:03Z" data-provider="edit_profile_alert_form_field" finished-at="2020-07-01T17:40:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paijo@mailinator.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[nickname]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Your Nickname Has Been Taken]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [yournicknamehasbeentaken] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [yournicknamehasbeentaken] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at object.Alert.byIdByTag(Alert.java:196)
	at testcases.Test_Edit_Profile.edit_profile_form_field_alert(Test_Edit_Profile.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- edit_profile_form_field_alert -->
        <test-method status="PASS" signature="done(org.testng.ITestResult)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="done" is-config="true" duration-ms="5504" started-at="2020-07-01T17:40:57Z" finished-at="2020-07-01T17:41:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=edit_profile_form_field_alert status=FAILURE method=Test_Edit_Profile.edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
        <test-method status="PASS" signature="set()[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="set" is-config="true" duration-ms="1216" started-at="2020-07-01T17:41:03Z" finished-at="2020-07-01T17:41:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method status="FAIL" signature="edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="edit_profile_form_field_alert" duration-ms="110302" started-at="2020-07-01T17:41:04Z" data-provider="edit_profile_alert_form_field" finished-at="2020-07-01T17:42:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paijo@mailinator.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[email]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[dikakoko04@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[please try again, email has been taken]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [pleasetryagainemailhasbeentaken] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [pleasetryagainemailhasbeentaken] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at object.Alert.byIdByTag(Alert.java:196)
	at testcases.Test_Edit_Profile.edit_profile_form_field_alert(Test_Edit_Profile.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- edit_profile_form_field_alert -->
        <test-method status="PASS" signature="done(org.testng.ITestResult)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="done" is-config="true" duration-ms="5411" started-at="2020-07-01T17:42:54Z" finished-at="2020-07-01T17:43:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=edit_profile_form_field_alert status=FAILURE method=Test_Edit_Profile.edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
        <test-method status="PASS" signature="set()[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="set" is-config="true" duration-ms="1261" started-at="2020-07-01T17:43:00Z" finished-at="2020-07-01T17:43:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method status="FAIL" signature="edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="edit_profile_form_field_alert" duration-ms="89888" started-at="2020-07-01T17:43:01Z" data-provider="edit_profile_alert_form_field" finished-at="2020-07-01T17:44:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paijo@mailinator.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[email]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[email.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Please Try Again Email Is Incorrect]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [pleasetryagainemailisincorrect] but found [noconnection!checkyournetworkorfindabettersignal!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [pleasetryagainemailisincorrect] but found [noconnection!checkyournetworkorfindabettersignal!]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at object.Alert.byIdByTag(Alert.java:196)
	at testcases.Test_Edit_Profile.edit_profile_form_field_alert(Test_Edit_Profile.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- edit_profile_form_field_alert -->
        <test-method status="PASS" signature="done(org.testng.ITestResult)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="done" is-config="true" duration-ms="5366" started-at="2020-07-01T17:44:31Z" finished-at="2020-07-01T17:44:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=edit_profile_form_field_alert status=FAILURE method=Test_Edit_Profile.edit_profile_form_field_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
        <test-method status="PASS" signature="set()[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="set" is-config="true" duration-ms="1255" started-at="2020-07-01T17:44:36Z" finished-at="2020-07-01T17:44:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method status="FAIL" signature="edit_profile_phone_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="edit_profile_phone_alert" duration-ms="108912" started-at="2020-07-01T17:44:38Z" data-provider="edit_profile_alert_Phone" finished-at="2020-07-01T17:46:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paijo@mailinator.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[phone-number]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[82278843303]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[please try again, phone has been taken]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [pleasetryagainphonehasbeentaken] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [pleasetryagainphonehasbeentaken] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at object.Alert.byIdByTag(Alert.java:196)
	at testcases.Test_Edit_Profile.edit_profile_phone_alert(Test_Edit_Profile.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- edit_profile_phone_alert -->
        <test-method status="PASS" signature="done(org.testng.ITestResult)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="done" is-config="true" duration-ms="5551" started-at="2020-07-01T17:46:26Z" finished-at="2020-07-01T17:46:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=edit_profile_phone_alert status=FAILURE method=Test_Edit_Profile.edit_profile_phone_alert(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
        <test-method status="PASS" signature="set()[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="set" is-config="true" duration-ms="1197" started-at="2020-07-01T17:46:32Z" finished-at="2020-07-01T17:46:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set -->
        <test-method status="PASS" signature="edit_profile_success_edit_fullname(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="edit_profile_success_edit_fullname" duration-ms="123892" started-at="2020-07-01T17:46:33Z" data-provider="edit_profile" finished-at="2020-07-01T17:48:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[paijo@mailinator.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dikakoko]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[fullname]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[myfullnameqcedit]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- edit_profile_success_edit_fullname -->
        <test-method status="PASS" signature="done(org.testng.ITestResult)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c]" name="done" is-config="true" duration-ms="5462" started-at="2020-07-01T17:48:37Z" finished-at="2020-07-01T17:48:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=edit_profile_success_edit_fullname status=SUCCESS method=Test_Edit_Profile.edit_profile_success_edit_fullname(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.Test_Edit_Profile@5b58ed3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
      </class> <!-- testcases.Test_Edit_Profile -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
